platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/utils/FileUtils.kt

```diff
@@ -12,9 +12,8 @@ class FileUtils {
    * Task that copies a file from the assets directory to a provided directory.
    * The asset's name is going to be kept in the new directory.
    */
-  class CopyFileFromAssetsTask(context: Context, listener: OnFileCopiedFromAssetsListener) : AsyncTask<String, Void, Boolean>() {
```

Comment:
Storing the listener this way when invoked from `MergeOfflineRegionsActivity` can leak the context that's carried with the anonymous inner class. The listener should be kept as a field in the `MergeOfflineRegionsActivity` and `CopyFileFromAssetsTask` should still keep a weak reference to it.

Reply:
you seem to be correct, I assumed this would result in a soft reference but it's not.